test/forcing/NetCDFPerFeatureDataProvider_Test.cpp

```diff
@@ -12,7 +12,7 @@
 #include <limits.h>
 #include <ctime>
 #include <time.h>
-
+#include <netcdf>
```

Comment:
Looking this file over, this shouldn't be necessary, but as you found, it was. I'm going to give it a try and see if I can figure out why.

Reply:
I won't hold up integration if I can't get a quick answer, though.